geometry_msgs/Pose current_pose
geometry_msgs/Pose target_pose
jaco_manipulation/BoundingBox bounding_box